src/cdk-experimental/menu/menu-group.ts

```diff
@@ -22,5 +22,15 @@ import {CdkMenuItem} from './menu-item';
 })
 export class CdkMenuGroup {
   /** Emits the element when checkbox or radiobutton state changed  */
-  @Output() change: EventEmitter<CdkMenuItem> = new EventEmitter();
+  @Output() change: EventEmitter<MenuItem> = new EventEmitter();
```

Comment:
Might be a TODO for a follow-up PR, but we typically want to create an event interface for change rather than emitting the related type (ideally ending with `Event`, e.g. `MenuItemChangeEvent` or `MenuItemSelectionChangeEvent`)